docs/source/yaml_basics.rst

```diff
@@ -0,0 +1,122 @@
+===========
+YAML Basics
+===========
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+YAML is a simple, human-readable data serialization format that is often used for specification files. The extensions
+of a file can be **.yaml** or **.yml**, both of which are accepted throughout the Vivarium framework.  The following
+are general rules to keep in mind when writing and interpreting Vivarium YAML files. Examples use snippets from Vivarium
+model configurations but do not go in-depth about that topic. For more information about model configurations, please
+see that section of this documentation.
+
+Structure
+---------
+
+YAML files are structured by lines and space indentations. Indentation levels should be either 2 or 4 spaces, and
+**tabs are not valid**.  For example, a configuration file that includes a ``BasePopulation()`` component would look
+like the following:
+
+.. code-block:: yaml
+
+    components:
+        vivarium_public_health:
+            population:
+                - BasePopulation()
+
+Comments
+--------
+
+YAML comments are denoted with the pound symbol ``#``, and can be placed anywhere, but must be separated from the
+preceding token by a space. For example, adding a comment to the configuration from above looks like this:
+
+.. code-block:: yaml
+
+    components:
```

Comment:
Still strongly advise against using the components block for examples.  What matters for users is the ``configuration`` block, as that passes into individual components via the builder and gets modified by the branches file.  The ``components`` section has meaningful semantics that are not yaml specific.

Reply:
I swapped this out , but not the list example because the only place I see those is when listing components. Configs are mappings as far as I can see